main_app/tests/test_app.py

```diff
@@ -19,3 +19,10 @@ def test_can_reformat_json_object(self):
         self.assertEqual(type(data), dict)
         self.assertEqual(data['userId'], 1)
         self.assertEqual(str(data.keys()), "dict_keys(['userId', 'id', 'title', 'completed'])")
+
+    def test_can_return_error(self):
+        """Can return an error when entering incorrect coordinate format"""
+        data = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
```

Comment:
I like what you've started but the URL is wrong. The URL here is only gonna test if that `jsonplaceholder` api is a dictionary. This test was originally built for us to reference. But again I like what you've started. If you want to keep trying to work at this, change the URL to match the URL we have in the API call and put the Python-pry on line 26 and start seeing what `data` is, then build the test that way

Reply:
thank you for your feedback!! Still at it!